src/view/point-event.js

```diff
@@ -1,37 +1,108 @@
-export const createPointEventTemplate = () => `
-  <div class="event">
-    <time class="event__date" datetime="2019-03-18">MAR 18</time>
-    <div class="event__type">
-      <img class="event__type-icon" width="42" height="42" src="img/icons/drive.png" alt="Event type icon">
-    </div>
-    <h3 class="event__title">Drive Chamonix</h3>
-    <div class="event__schedule">
-      <p class="event__time">
-        <time class="event__start-time" datetime="2019-03-18T14:30">14:30</time>
-        &mdash;
-        <time class="event__end-time" datetime="2019-03-18T16:05">16:05</time>
+import { getTotalPrice } from '../utils/useRender';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+dayjs.extend(duration);
+
+export const createPointEventTemplate = (pointEvent) => {
+  const { type, destination, dateFrom, dateTo, basePrice, offers, isFavorite } = pointEvent;
+
+  const totalPrice = getTotalPrice(offers, basePrice);
+
+  const startDay = dayjs(dateFrom).format('MMM D');
+
+  const startTime = dayjs(dateFrom).format('HH:mm');
+  const startDate = dayjs(dateFrom).format('YYYY-MM-DD');
+
+  const finishTime = dayjs(dateTo).format('HH:mm');
+  const finishDate = dayjs(dateTo).format('YYYY-MM-DD');
```

Comment:
```suggestion
  const startDay = dayjs(dateFrom).format('MMM D');
  const startTime = dayjs(dateFrom).format('HH:mm');
  const startDate = dayjs(dateFrom).format('YYYY-MM-DD');
  const finishTime = dayjs(dateTo).format('HH:mm');
  const finishDate = dayjs(dateTo).format('YYYY-MM-DD');
```

и форматы в отдельные константы, а еще лучше заведи перечисление с форматами